assembl/graphql/schema.py

```diff
@@ -997,7 +997,9 @@ def resolve_ideas(self, args, context, info):
         root_idea_id = discussion.root_idea.id
         descendants_query = model.get_descendants_query(
             root_idea_id, inclusive=True)
-        query = query.filter(model.id.in_(descendants_query)
+        query = query.outerjoin(
```

Comment:
I suspect that this is not fully right. There are many idea links that have both a null source and a null target, or a source and null target. But I think it's correct that the order is an ascending number. @maparent Maybe could chime in?

Reply:
@ayyazdaniaryan I do exactly the same thing that in the rest api. It's a left outer join, and the order by is on both IdeaLink.order and Idea.creation_date

Reply:
Alright. All good with me then.

Reply:
In the rest api : https://github.com/assembl/assembl/blob/68f3348cfe35de7b962352303b61e893b26584a6/assembl/views/api/idea.py#L125-L130